### PR TITLE
Update "Getting involved" documentation page.

### PR DESCRIPTION
**Pull request by MichaelGrupp**
Some things were outdated after recent changes.

Signed-off-by: Michael Grupp <grupp@magazino.eu>

### CODE REVIEW

File: docs/source/getting_involved.rst [hunk 1]
```diff
@@ -19,18 +19,11 @@ Getting involved
...
-The slides are also made available after each session but there is no video recording.
-
-If you want to stay tuned with announcements (such as new major releases or new open house sessions), you can join `the Cartographer mailing list`_ although you can not interact with this mailing list anymore.
+If you want to stay tuned with announcements (such as new major releases), you can join `the Cartographer mailing list`_ although you can not interact with this mailing list anymore.
 
 .. _the Cartographer mailing list: https://groups.google.com/forum/#!forum/google-cartographer
 
-If you think you've found an issue in Cartographer, you are welcome to open a `GitHub issue`_ but don't forget to provide a way to reproduce your bug!
-Typically, join a ``.bag`` and a link to a fork of the ``cartographer_ros`` repository containing your configuration and launch files.
+If you think you've found an issue in Cartographer, you are welcome to open a `GitHub issue`_.
```
Comment (MichaelGrupp) [inline docs/source/getting_involved.rst:26]: handled by the issue templates

File: docs/source/getting_involved.rst [hunk 2]
```diff
@@ -19,18 +19,11 @@ Getting involved
...
 There are multiple ways to get involved!
 
-Twice a month, the project hosts "Open House Hangouts" sessions that are essentially meetings open to everyone to join in.
-The call typically recaps the recent and ongoing development around Cartographer and Cartographer ROS.
-The developers are then open to questions from the community, this is a great time to ask about contribution ideas.
-If you don't feel like talking or being seen, you are free to join anyway and skulk!
-The slides are also made available after each session but there is no video recording.
-
-If you want to stay tuned with announcements (such as new major releases or new open house sessions), you can join `the Cartographer mailing list`_ although you can not interact with this mailing list anymore.
+If you want to stay tuned with announcements (such as new major releases), you can join `the Cartographer mailing list`_ although you can not interact with this mailing list anymore.
```
Comment (MichaelGrupp) [inline docs/source/getting_involved.rst:22]: @wohe @bochen87 I would keep it like this. Or did you have other plans for the mailing list that I don't know?

Comment (bochen87) [reply docs/source/getting_involved.rst:22]: no, we didn't - maybe to revive more active involvement from the community, we could do some more regular sessions? like monthly or bi-monthly?

**Review by bochen87 — APPROVED** [review]

File: docs/source/getting_involved.rst (hunk 2 above)
Comment (wohe) [reply docs/source/getting_involved.rst:22]: Regarding the mailing list: I do not think anyone of us has admin access to the old group and it does not seem there has been anything happening there in the last years. Maybe we give it up and point people towards GitHub issues where they actually have a chance to get some info/answers?

Comment (MichaelGrupp) [reply docs/source/getting_involved.rst:22]: yes, I think this makes the most sense.

**Pull request merged**